pkg/kubelet/oom/oom_watcher_linux.go

```diff
@@ -73,15 +73,15 @@ func (ow *realWatcher) Start(ref *v1.ObjectReference) error {
 
 		for event := range outStream {
 			if event.VictimContainerName == recordEventContainerName {
-				klog.V(1).Infof("Got sys oom event: %v", event)
+				klog.V(1).InfoS("Got sys oom event", "event", event)
 				eventMsg := "System OOM encountered"
 				if event.ProcessName != "" && event.Pid != 0 {
 					eventMsg = fmt.Sprintf("%s, victim process: %s, pid: %d", eventMsg, event.ProcessName, event.Pid)
 				}
 				ow.recorder.Eventf(ref, v1.EventTypeWarning, systemOOMEvent, eventMsg)
 			}
 		}
-		klog.Errorf("Unexpectedly stopped receiving OOM notifications")
+		klog.ErrorS(fmt.Errorf("Unexpectedly stopped receiving OOM notifications"), "")
```

Comment:
```suggestion
		klog.ErrorS(nil, "Unexpectedly stopped receiving OOM notifications")
```

Reply:
@ehashman if err is nil, may be InfoS more available?